ticketutil/bugzilla.py

```diff
@@ -1,11 +1,24 @@
 import logging
+import os
 
 import requests
 
+import base64
+
 from . import ticket
 
 __author__ = 'dranck, rnester, kshirsal'
 
+# Disable warnings for requests because we aren't doing certificate verification
```

Comment:
You can remove lines 12-21. This is handled in ticket.py now, which all the other tool-specific files import.